spec/support/requirements_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+%i[form summary].each do |style|
+  RSpec::Matchers.define :"have_#{style}_item" do |field, key, context = {}|
```

Comment:
I'm wondering if we're actually testing the presence of a message rather than an item. The expectation we do is against a message we construct, so there is no way to test the wider properties of the item.

Reply:
I see what you mean: we're only testing the 'text' key in a hash that can contain other pairs, like 'href'. However, I don't think we actually want to test any of those here, since they're characteristics that are decided by the callers; we could test the Issue class to ensure it supports them, but that's not really in scope here.